frontend/packages/signless-transactions/src/hooks/use-create-session.ts

```diff
@@ -102,16 +102,34 @@ function useCreateSession(programId: HexString, metadata: ProgramMetadata | unde
     const { signer } = await web3FromSource(account.meta.source);
     const { signRaw } = signer;
 
+    console.log('SIGNATURE:');
+    console.log('METADATA');
+    console.log(metadata);
+    console.log('ACCOUNT');
+    console.log(account);
+    console.log('PAYLOAD_TO_SIGN');
+    console.log(payloadToSign);
+    console.log('web3FromSource');
+    console.log(web3FromSource);
+    console.log('SIGNER');
+    console.log(signer);
+    console.log('signRaw');
+    console.log(signRaw);
+
     if (!signRaw) {
       throw new Error('signRaw is not a function');
     }
 
-    if (!metadata.types?.others?.output) {
+    if (metadata.types?.others?.output === null) {
```

Comment:
if (metadata.types?.others === undefined) you will not throw error

Reply:
it can't be undefined, it's just a number or null